examples/text_to_image/train_text_to_image.py

```diff
@@ -421,7 +452,14 @@ def main():
         )
     logging_dir = os.path.join(args.output_dir, args.logging_dir)
 
-    accelerator_project_config = ProjectConfiguration(total_limit=args.checkpoints_total_limit)
+    save_iteration = get_save_iteration(args.output_dir)
+
+    accelerator_project_config = ProjectConfiguration(
+        total_limit=args.checkpoints_total_limit,
+        automatic_checkpoint_naming=True,
+        project_dir=args.output_dir,
+        iteration=save_iteration,
```

Comment:
args to allow accelerate to handle checkpoint naming

Reply:
For my curiosity, we're resorting to `automatic_checkpoint_naming` because that it will be easier to manage the checkpointing limits and resuming from the checkpoints?